pages/implementation/avo-functions-overview.mdx

```diff
@@ -7,11 +7,15 @@ import Link from '../../components/Link';
 
 ## <a name="avo-generated-file"></a> Avo generated file containing Avo Functions
 
-The Avo file is the code representation of the tracking plan you define in Avo for a given source. You can think of a source as a platform that sends events. For example if you set up an iOS/Swift source the generated file would be a `.swift` file. Since the tracking plans are different for each customer the content of this file is also different. We generate the files and you can download them either with <Link passHref href="/implementation/cli"><a>the Avo CLI</a></Link> or in the Avo web interface in <Link passHref href="/workspace/implement"><a>the Implement tab</a></Link>.
+The Avo generated code is the representation of the tracking plan you define in Avo for a given source. You can think of a source as a platform that sends events. 
+For example if you set up an iOS/Swift source the generated code will be in a `.swift` file. 
+Since the tracking plans are different for each customer the content of this file is also different. 
+We generate the files and you can download them either with <Link passHref href="/implementation/cli"><a>the Avo CLI</a></Link> or in the Avo web interface in <Link passHref href="/workspace/implement"><a>the Implement tab</a></Link>.
```

Comment:
Any change here other than the line breaks? 👀  Don't we want to keep this in a single paragraph?

Reply:
Yes, that's only for readability in the editor. Single line break does not change the output, to get a new paragraph we need a double line break